modMcf/src/org/aion/mcf/trie/TrieImpl.java

```diff
@@ -876,16 +871,10 @@ private void traceTrie(byte[] stateRoot, ScanAction action) {
 
     public boolean validate() {
         synchronized (cache) {
```

Comment:
inside the scanTreeLoop already synchronized the cache, Can we remove the synchronized in this method?

Reply:
This `validate` method is actually not used either. Should I completely remove it?

Reply:
if we confirmed the method is thread-safe. we should remove it.
The core components will affect some performance of the kernel.

Reply:
Remove the extra synchronization and commented out the method. It can be re-enabled and used correctly if ever needed.